content/blog/intro-to-web-accessability/index.md

```diff
@@ -0,0 +1,301 @@
+- `3rem` will be `48px` (3 * 16px)
+
+Likewise, when the browser's font size is set to `20px`:
+
+- `1rem` will be `20px` (1 * 20px)
+- `1.5rem` will be `30px` (1.5 * 20px)
+- `3rem` will be `60px` (3 * 20px)
+
+> Something to keep in mind is that `rem` is a _relative_ font size. It's relative to the root element's font size. _This means that you cannot set a default `px` value font size in `:root`  , as it will disable font scaling, even if the rest of your page is using `rem` values._
+
+It's highly encouraged to keep font sizes standardized around the `1rem` value. While this may seem like a frustrating limitation at first, see it from the user's perspective.
+
+Say site "A" sets their font size to `1rem`, and site "B" sets their font size to `0.8rem`. When the user switches from "A" to "B", the font size drastically decreases, requiring the user to change their font size. Then, when they switch back to site "A", they're left with too large of font size. By respecting the user's setting of font size, you're ensuring that their experience jumping from site to site is more consistent and a nicer experience.
+
+Want to learn more about `rem` and font sizing? [Take a look at this in-depth blog post that covers even more](https://www.24a11y.com/2019/pixels-vs-relative-units-in-css-why-its-still-a-big-deal/).
+
+# Keyboard is King {#keyboard}
+
+Just as developers have preferences with keyboard or mouse, so too do your end-users. Some users may only be able to utilize the keyboard to navigate the digital world. Not only is keyboard navigation critical for accessibility, but it enables power users of your application to be more efficient as well.
+
+While you may immediately think of "keyboard shortcuts" to trigger different applications, it's often easy to forget about smaller interactions as well.
+
+> This isn't to say that keyboard shortcuts aren't helpful - they can serve as shorthand for operations that might otherwise take considerable more effort using only the keyboard. Do consider implementing them in your app when they make sense.
+
+Consider a site with a large number of navigation links like [The New York Times](https://www.nytimes.com/).
+
+![The initial homepage of New York Times](./nyt_initial.png)
+
+Their site has 30+ individual items in their header that you'd need to tab through in order to get to the main portion of the website. Safe to say that most users of the site are likely to want to access that main content quickly. Additionally, users that rely on a screen reader that aren't familiar with the site's layout wouldn't know when to stop tabbing without slogging through every item in the header.
+
+As such, many sites (including New York Times) include a "Skip to Content" button that doesn't become visible until you've tabbed into it. Unless you were navigating the site by keyboard, you wouldn't know it was there.
+
+![A "skip to content" button shown on the New York Times' site](./nyt_skip.png)
```

Comment:
This is very small, maybe include a suggestion about using the lightbox or including a zoomed version?